sessionctx/variable/sysvar.go

```diff
@@ -731,6 +732,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal | ScopeSession, TiDBCapturePlanBaseline, "0"},
 	{ScopeGlobal | ScopeSession, TiDBUsePlanBaselines, BoolToIntStr(DefTiDBUsePlanBaselines)},
 	{ScopeGlobal | ScopeSession, TiDBIsolationReadEngines, "tikv,tiflash"},
+	{ScopeSession, TiDBStoreLimit, strconv.FormatUint(atomic.LoadUint64(&config.GetGlobalConfig().TiKVClient.StoreLimit), 10)},
```

Comment:
`TiDBStoreLimit` should be a global scope variable(not session)

Reply:
Addressed.